backend/dal/Migrations/Initial/Up/PreUp/01-History.sql

```diff
@@ -815,7 +815,7 @@ CREATE TABLE [dbo].[PIMS_USER_ORGANIZATION_HIST] (
     [USER_ORGANIZATION_ID] BIGINT NOT NULL,
     [ORGANIZATION_ID] BIGINT NOT NULL,
     [USER_ID] BIGINT NOT NULL,
-    [ROLE_ID] BIGINT NOT NULL,
```

Comment:
According to Doug's scripts this should remain as `NOT NULL`

Reply:
Yeah, this isn't supported by the frontend, so I'm going to try and get him to change this.

Reply:
I'll let you schedule a discussion.  I can update my PR once we have confirmation.
You could always just assume the role the user asking for is the same role that would be applied here.

Reply:
Right, it just gets complicated:
1) when there are multiple roles.
2) when the user updates their organization.

Of course it is possible to make this work, I just would rather not spend time supporting database functionality that we aren't using.